autoload/qfedit.vim

```diff
@@ -50,7 +50,9 @@ function! qfedit#line(item) abort
         \ (get(a:item, 'end_col') && a:item.col != a:item.end_col ? '-' . a:item.end_col : '')
         \ : '') .
         \ qfedit#type(a:item.type, a:item.nr) . '| ' .
-        \ substitute(substitute(a:item.text, '\v^%(\t| )+', '', ''), '\v\n%(\n|\t| )*', ' ', 'g'))[:1023]
+        \ (get(a:item, 'valid', v:false) ? substitute(substitute(a:item.text, '\v^%(\t| )+', '', ''), '\v\n%(\n|\t| )*', ' ', 'g')
```

Comment:
Is there any case `a:item` doesn't have this key ([ref](https://github.com/vim/vim/blob/v7.4/src/quickfix.c#L3784))? I think it can simplified to `a:item.valid`.

Reply:
Okay, fixed there.